modules/libcom/test/fdmgrTest.c

```diff
@@ -119,7 +119,7 @@ int main (int argc, char **argv)
          SEVCHK(ca_search (argv[1], &chan), NULL);
     }
 
-    while (1) {
+    for(int loop=0; loop < 5; loop++)  {
```

Comment:
Interesting… This performance measurement program has not been built by default since 2006. It's not obvious what it actually does, but it does need a PV name argument to exercise all its functionality. It is also not build-compatible with the 7.0 branch, because despite living in the module/libcom/test directory it uses the cadef.h header from the ca module (which isn't built until after libcom/test, so if we enabled it in the Makefile the build would fail).

Your change at least makes the program exit so I'd be happy for us to apply it, but I don't think we can enable it in the build until someone further evaluates its usefulness and maybe moves it into modules/ca/test (which doesn't exist yet).